trustmanager/x509filestore_test.go

```diff
@@ -82,6 +124,21 @@ func TestAddCertFromFileX509FileStore(t *testing.T) {
 	}
 }
 
+// TestNewX509FileStoreEmpty verifies the behavior of the Empty function
+func TestNewX509FileStoreEmpty(t *testing.T) {
+	tempDir, err := ioutil.TempDir("", "cert-test")
+	assert.NoError(t, err)
+	defer os.RemoveAll(tempDir)
+
+	store, err := NewX509FileStore(tempDir)
+	assert.NoError(t, err, "failed to create a new X509FileStore: %v", store)
+	assert.True(t, store.Empty())
+
+	err = store.AddCertFromFile("../fixtures/root-ca.crt")
+	assert.NoError(t, err, "failed to add certificate from file")
```

Comment:
I just noticed some of the other tests in notary were were passing strings to assert functions, to state why things were failing.  I wasn't sure whether the standard practice was to include strings about what was happening when the error occurred, or no.  I'm happy to take them out.